src/ensembl/src/content/app/custom-download/CustomDownload.tsx

```diff
@@ -1,22 +1,45 @@
-import React, { FunctionComponent } from 'react';
+import React, { FunctionComponent, useEffect } from 'react';
 import { connect } from 'react-redux';
 import PreFilterPanel from 'src/content/app/custom-download/containers/pre-filters-panel/PreFilterPanel';
-import { getShowPreFilterPanel } from './state/customDownloadSelectors';
+import {
+  getShowPreFilterPanel,
+  getCustomDownloadActiveGenomeId
+} from './state/customDownloadSelectors';
 import { RootState } from 'src/store';
 import CustomDownloadHeader from './containers/header/CustomDownloadHeader';
 import CustomDownloadContent from './containers/content/CustomDownloadContent';
+import CustomDownloadAppBar from './containers/app-bar/CustomDownloadAppBar';
+import { setActiveGenomeId } from './state/customDownloadActions';
+import { getCommittedSpecies } from '../species-selector/state/speciesSelectorSelectors';
+import { CommittedItem } from '../species-selector/types/species-search';
```

Comment:
Perhaps absolute paths starting with src instead?

Reply:
I was using the auto import feature to import these.
I am starting to think why we shouldn't implement an index file for types somewhere in the root level as it would make importing a lot easier. In the example above, it's not very easy to keep in mind where `CommittedItem` is defined. We also do not have a guarantee that the name `CommittedItem` will never be re-used on other apps.
Wouldn't it be better if we can use: ` import {CommittedItem} from 'src/types'` ? Any thoughts?

Reply:
> We also do not have a guarantee that the name CommittedItem will never be re-used on other apps.

Regarding this point, it would be much easier if we try to avoid it, because otherwise it will get harder to find the correct thing. Particularly when something breaks in the interface, and you need to find out which component the broken piece corresponds to. Also, error messages will contain the names of components/files. So, would really, really like us to not reuse the same name for different things in different parts of our application.

As for the index-like `src/types` file, I am still not sold on it, to be honest. Perhaps I am just not using relevant editor's features, and so never found myself wanting this.

Reply:
> So, would really, really like us to not reuse the same name for different things in different parts of our application.

That's what I am thinking as well. Once the application grows, it would become harder to keep in mind if the name is already used.

I've currently updated my vscode settings to use absolute path when I do an auto import.